### PR TITLE
PIM-4996: Translate dates in the UI in grid screens

### PR DESCRIPTION
**Pull request by skeleton**
| Q                 | A
| ----------------- | ---
| Specs             | :white_check_mark: 
| Behats            | :white_check_mark: 
| Blue CI           | ?
| Changelog updated | :white_check_mark: 
| Review and 2 GTM  | :white_check_mark: 
| Micro Demo (PO)   | TODO
| Migration script  | N
| Tech Doc          | N

### CODE REVIEW

File: CHANGELOG-1.5.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # 1.5.x
 
+## Bug fixes
```
Comment (ClementGautier) [inline CHANGELOG-1.5.md]: Wrong section ? ( We used "BC break" label for other versions )

**Comment by ClementGautier** [comment]
tiny comment, GTM :+1:

**Comment by juliensnz** [comment]
GTM

File: CHANGELOG-1.5.md (hunk 1 above)
Comment (skeleton) [inline CHANGELOG-1.5.md]: yep, good catch :)

**Pull request merged**